.github/workflows/kind.yml

```diff
@@ -13,6 +13,7 @@ on:
 
 env:
   KIND_VERSION: v0.20.0
+  SKIP_CASES: "TestAntctl|TestBandwidth|TestBasic|TestBatchCreatePods|TestFlowAggregator/IPv4/Antctl|TestFlowAggregator/IPv6/Antctl|TestSecurity|TestSupportBundleController/testSupportBundle|TestAntreaApiserverTLSConfig"
```

Comment:
This is very impractical IMO, because now instead of having the ability to declare an e2e test case as "less important" as part of its definition, we have to go here and keep updating this list.

While I agree that we need to stop defining a custom `skip` flag to avoid conflicts with the standard one, we should probably consider _renaming_ the flag instead of _removing_ it.

Reply:
sure, would do that